yfinance/utils.py

```diff
@@ -345,6 +347,14 @@ def parse_actions(data):
 
             dividends.columns = ["Dividends"]
 
+        if is_capital_gains_data_supported and "capitalGains" in data["events"]:
```

Comment:
Only check if "capitalGains" in data["events"]

Reply:
Sure